library/src/test/java/com/hannesdorfmann/adapterdelegates3/DiffDelegationAdapterTest.java

```diff
@@ -0,0 +1,53 @@
+package com.hannesdorfmann.adapterdelegates3;
+
+import android.support.v7.recyclerview.extensions.AsyncListDiffer;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+/**
+ * Created by six_hundreds on 28.06.18.
+ */
+@RunWith(RobolectricTestRunner.class)
```

Comment:
If possible I would like to keep Robolectirc out of this library. From experience I know it adds just some extra maintainance costs and overall it looks like it adds very little value to have the following tests compared to the maintainance cost. What do you think?

Reply:
Yeah, i know about that costs. But ListAdapter internally uses Looper, which can't be mocked without Robolectric.

Reply:
I'm done with fixes. So what about tests? We can move out Robolectric, but that's mean that we will not be able to work with ListAdapter and basically not be able to test it somehow